### DISCUSSION TITLE
デバイスの名前どうしよう？

### DISCUSSION THREAD

**Discussion by matsujirushi** [algyan/XIAOGYAN#21] [General]
基板という部品ではなく、もっと広い範囲をカバーしたものを提供したくて、とりあえず「IoT ALGYAN 8周年 オリジナルIoTデバイス」と呼んでいますが、、、
もっと気の利いた名前ないでしょうか？？

**Comment by atkogure** [comment]
なるほどです。


例えば「PinKit」のように固有名詞を作っても良いですね。
しかし「ジョージ8」「松岡8号」のようにアレンジしすぎると意味が伝わりませんよね。
とりあえず、
「ALGYAN」
「8周年」
「IoTデバイス」
の要素を含ませたほうが無難かも？

とすると・・・、
ALGYAN IoT Edge No.8
ALGYAN IoT Edge 8th
ALGYAN IoT Edge 2023

うーん、平凡かなw

少しひねって・・・、
ALGYAN Edge 8　(あるじゃねっじ えいと)　→ブラウザのedgeと混同される？ 
ALGYAN Endpoint 8　(あるじゃん エンドポイントえいと)　→結構気に入ってます。

うーん、頭固いかなぁ・・・。

皆様のご意見も頂きたいです♫

**Reply by y-ohe** [reply]
ALGYANでプロデュースするXIAOを載せるボードなので、
XIAOGYAN
とかは？

**Reply by matsujirushi** [reply]
@atkogure @y-ohe 
ネタありがとうございます！
ライブラリの作成に着手するので、そろそろ決めなければ（汗

そうですね、、、呼びやすく短くて覚えやすいものにしたいです。（なので、たしかに意味は伝わらない文言になりますね）

`XIAOGYAN`を強引に短くして、`XIAON`とかどうだろう？

**Reply by atkogure** [reply]
XIAON、語感が良さげですね。
ただ、「XIAON」でググると有象無象がたくさん出てきて、商標的に問題ありそうです。
あと、スーパーマーケットみたいだし、ALGYANの痕跡が無さすぎると存じますw
やめましょうw


あとは、
「あるじゃお」とか？
ALXIAO
ALGYAO
・・・、うーん、苦しいなぁ。
語感も悪いですね。


そう考えると、大栄さんの「XIAOGYAN(しゃおじゃん)」良きですね！

いかがでしょう。
